README.md

```diff
@@ -1 +1,182 @@
-# Purple-Coders---HTML-CSS-JavaScript-capstone-project
\ No newline at end of file
+
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+ 
+  <img src="./images/purpleLogoBg.png" alt="logo" width="70%"  height="400px"/>
+  <br/>
+
+  <h3><b>Purple-Coders Event </b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+      - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Portfolio: Pop up window
+] <a name="about-project"></a>
+
+PCapstone Project(1): Purple-Coders is a Its a project meant to master my skills in version control, GitHub flow, pull requests in GitHub and JavaScript.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- HTML & CSS
+- Linters
+- Git & Github
+- JavaScript
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+
+- **[Pop up window]**
+- **[contact form storage capability]**
+- **[addition of logo rotate on hover]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+## Getting Started
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://kel-nana.github.io/Purple-Coders---HTML-CSS-JavaScript-capstone-project/)
```

Comment:
- [ ] Great job deploying your site, kindly a record video presenting the features of the project and add the link to the video to your pull request description.
The maximum length of the video is 5 minutes.
The video contains a clear and objective description about the project in professional English. The topic and purpose of the project are made clear.
The video contains a demo of the project, showing the pages in different screen sizes.
The video contains a highlight about a piece of the code the student finds interesting or is proud of making.

Reply:
- [ ] This issue is well resolved :heavy_check_mark: